src/scout_apm/core/traceback.py

```diff
@@ -0,0 +1,21 @@
+# Module for helper functions to capture tracebacks
+
+import traceback
+
+
+def capture():
+    stack = traceback.extract_stack()
```

Comment:
We may want to pass a `limit` parameter with a reasonable value. I [ran this script](https://gist.github.com/JettJones/c236494013f22723c1822126df944b12) to benchmark the performance impact of fetching the stack. This has a low limit value (2), so ours wouldn't be as fast. We need a deeper stack.